### PR TITLE
Rebase/element android 1.4.13

### PR DESCRIPTION
**Pull request by Claire1817**
#600

### CODE REVIEW

File: vector/src/main/java/im/vector/app/core/di/SingletonModule.kt [hunk 1]
```diff
@@ -125,12 +125,8 @@ object VectorStaticModule {
                 applicationFlavor = BuildConfig.FLAVOR_DESCRIPTION,
                 roomDisplayNameFallbackProvider = vectorRoomDisplayNameFallbackProvider,
                 threadMessagesEnabledDefault = vectorPreferences.areThreadMessagesEnabled(),
-<<<<<<< HEAD
-                presenceSyncEnabled = BuildConfig.PRESENCE_SYNC_ENABLED,
```
Comment (yostyle) [inline vector/src/main/java/im/vector/app/core/di/SingletonModule.kt:127]: You keep PRESENCE_SYNC_ENABLED config in gradle file but you don't use it here anymore. 
Why ?

File: vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt [hunk 2]
```diff
@@ -186,22 +186,13 @@ class HomeActivityViewModel @AssistedInject constructor(
...
-                                checkBootstrap = true
-                            }
-                        }
-                        is SyncStatusService.Status.Idle        -> {
-                            updateIdentityServer()
-=======
                         is SyncStatusService.Status.InitialSyncProgressing -> {
-                            // Schedule a check of the bootstrap when the init sync will be finished
-                            checkBootstrap = true
+                                if (BuildConfig.ENABLE_CROSS_SIGNING) {
```
Comment (yostyle) [inline vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt:190]: Please add a tchap comment here.

File: vector/src/main/java/im/vector/app/features/home/room/detail/timeline/action/MessageActionsViewModel.kt [hunk 3]
```diff
@@ -455,16 +455,13 @@ class MessageActionsViewModel @AssistedInject constructor(
                                  messageContent: MessageContent?,
                                  actionPermissions: ActionPermissions): Boolean {
         // We let reply in thread visible even if threads are not enabled, with an enhanced flow to attract users
-<<<<<<< HEAD
+
         // Tchap: don't show the canReplyInThread quick action if it's not enable in the labs
         if (!vectorPreferences.areThreadMessagesEnabled()) return false
-=======
-//        if (!vectorPreferences.areThreadMessagesEnabled()) return false
```
Comment (yostyle) [inline vector/src/main/java/im/vector/app/features/home/room/detail/timeline/action/MessageActionsViewModel.kt]: Do you know why this line is commented in EA ?

File: vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MergedRoomCreationItem.kt [hunk 4]
```diff
@@ -140,16 +140,10 @@ abstract class MergedRoomCreationItem : BasedMergedItem<MergedRoomCreationItem.H
...
         if (isDirect) {
-<<<<<<< HEAD
             // Tchap: Hide the domain if we are in DM.
             holder.roomDescriptionText.text = holder.view.resources.getString(
                     R.string.this_is_the_beginning_of_dm,
                     roomDisplayName
-=======
-            holder.roomDescriptionText.text = holder.view.resources.getString(
-                    R.string.this_is_the_beginning_of_dm,
-                    distinctMergeData.lastOrNull()?.memberName ?: ""
```
Comment (yostyle) [inline vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MergedRoomCreationItem.kt]: should we use this data to compute the roomDisplayName ?

File: vector/src/main/java/im/vector/app/core/di/SingletonModule.kt (hunk 1 above)
Comment (Claire1817) [reply vector/src/main/java/im/vector/app/core/di/SingletonModule.kt:127]: they don't use it in Element anymore.
I will delete it in the gradle file

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -138,7 +136,6 @@ And add `tools:ignore="UnusedResources"` to the string, to let lint ignore that
 
 The string will be removed during the next sync with Weblate.
```
[the remark this answers is not in the document]
Comment (giomfo) [reply CONTRIBUTING.md:137]: @Claire1817 you can remove these 2 sections

**Pull request merged**